jose/jwk.py

```diff
@@ -138,5 +138,5 @@ def to_dict(self):
         return {
             'alg': self._algorithm,
             'kty': 'oct',
-            'k': base64url_encode(self.prepared_key),
+            'k': base64url_encode(self.prepared_key).decode('ascii'),
```

Comment:
Nitpick: I know that the encoding names get normalized anyway, but I'd prefer to have one style. `ASCII` is capitalized everywhere else, so I'd like it capitalized here as well.